DEVELOPMENT.md

```diff
@@ -808,7 +804,36 @@ We use [Cypress](https://cypress.io) for running frontend-based end-to-end tests
 
 #### Running frontend end-to-end tests locally
 
-Make sure to [build the frontend assets](#building-the-frontend) and run `tools/bin/mage dev:initStack dev:sqlDump` to create a seed database which the tests can reset the database to in between runs. To run the stack when working on end-to-end tests, use the `tools/bin/mage dev:startDevStack` command. This will run the runs The Things Stack with proper configuration for the end-to-end tests. Note: this command does not output anything, but logs are written to `.cache/devStack.log`.
+The preferred way to run the end-to-end tests is to use the interactive launcher tool with:
+
+```bash
+$  tools/bin/mage dev:serveDevWebui
+```
+
+1. Make sure to run `tools/bin/mage dev:dbStop dev:dbErase dev:dbCreate dev:dbStart dev:initStack dev:sqlDump dev:sqlCreateSeedDb` to create a seed database which the tests can reset the database to in between runs.
```

Comment:
Should you do this before or after `dev:serveDevWebui`?

Also how often do you need to do this normally?

Reply:
I don't seem to find the mage target `dev:sqlCreateSeedDb`. Where's this?

Reply:
> Also how often do you need to do this normally?

normally I do this once when starting working on something

Reply:
> I don't seem to find the mage target dev:sqlCreateSeedDb. Where's this?

I think it does not exist, my bad